ScrollViewer.xaml

```diff
@@ -1,812 +1,323 @@
-    <!-- Default style for Windows.UI.Xaml.Controls.Primitives.ScrollBar -->
-    <Style TargetType="ScrollBar">
-        <Setter Property="MinWidth" Value="{ThemeResource ScrollBarSize}" />
-        <Setter Property="MinHeight" Value="{ThemeResource ScrollBarSize}" />
-        <Setter Property="Background" Value="{ThemeResource ScrollBarBackground}" />
-        <Setter Property="Foreground" Value="{ThemeResource ScrollBarForeground}" />
-        <Setter Property="BorderBrush" Value="{ThemeResource ScrollBarBorderBrush}" />
-        <Setter Property="IsTabStop" Value="False" />
-        <Setter Property="Template">
-            <Setter.Value>
-                <ControlTemplate TargetType="ScrollBar">
-                    <Grid x:Name="Root">
-
-                        <Grid.Resources>
-                            <ControlTemplate x:Key="RepeatButtonTemplate" TargetType="RepeatButton">
-                                <Grid x:Name="Root" Background="Transparent">
-                                    <VisualStateManager.VisualStateGroups>
-                                        <VisualStateGroup x:Name="CommonStates">
-                                            <VisualState x:Name="Normal" />
-                                        </VisualStateGroup>
-                                    </VisualStateManager.VisualStateGroups>
+<Style TargetType="ScrollBar" x:Key="HorizontalScrollBar">

```

Comment:
Note that this is specialized to horizontal now instead of having the elements for both horizontal and vertical